src/app/store/vlan/types/base.ts

```diff
@@ -20,6 +21,7 @@ export type BaseVLAN = TimestampedModel & {
   relay_vlan: VLAN[VLANMeta.PK] | null;
   secondary_rack: Controller[ControllerMeta.PK] | null;
   space: Space[SpaceMeta.PK] | null;
+  subnet_ids: Subnet["id"][];
```

Comment:
Could this ever be null?

Reply:
Nope, if there's no subnets on a VLAn then that property will just be an empty list.